File: 01.JS-Basic/More Exercises/PB - More Exercises/Conditional Statements/Firm.js
function firm(input) {

    let neededHours = Number(input[0])
    let days = Number(input[1])
    let employees = Number(input[2])

    let workingDays = days * 0.9
    let workingHours = Math.floor(workingDays * 8) + employees * days * 2

    if (workingHours >= neededHours) {
        console.log(`Yes!${workingHours - neededHours} hours left.`)
    } else { console.log(`Not enough time!${neededHours - workingHours} hours needed.`) }

}
firm(["50", "5", "2"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatements/ConditionalStatementsMoreExercises/Firm.java
package ConditionalStatements.ConditionalStatementsMoreExercises;

import java.util.Scanner;

public class Firm {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int projectHours = Integer.parseInt(scan.nextLine());
        int days = Integer.parseInt(scan.nextLine());
        int peopleOvertime = Integer.parseInt(scan.nextLine());

        int workingHours = (int) ((days * 8) * 0.9 + (2 * peopleOvertime * days));

        if (projectHours <= workingHours) {
            System.out.printf("Yes!%d hours left.", workingHours - projectHours);
        } else {
            System.out.printf("Not enough time!%d hours needed.", projectHours - workingHours);
        }
    }
}
